Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -661,6 +661,11 @@ private void SurfaceObserver_OnDataReady(SurfaceData cookedData, bool outputWrit
                     meshObject.Renderer.sharedMaterial = (displayOption == SpatialAwarenessMeshDisplayOptions.Visible) ?
```

Comment:
We have a third mesh observer that might be good to update with this change too, for the new XR SDK pipeline:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/5c29abeba9e4dd895951dc681051e0dc6343fd4f/Assets/MRTK/Providers/XRSDK/GenericXRSDKSpatialMeshObserver.cs#L494-L496